symbolic-debuginfo/src/wasm.rs

```diff
@@ -273,14 +273,14 @@ impl<'data> Dwarf<'data> for WasmObject<'data> {
 /// An iterator over symbols in the WASM file.
 ///
 /// Returned by [`WasmObject::symbols`](struct.WasmObject.html#method.symbols).
-pub struct WasmSymbolIterator<'data> {
-    funcs: Vec<Symbol<'data>>,
+pub struct WasmSymbolIterator<'data, 'object> {
+    funcs: Box<dyn Iterator<Item = Symbol<'data>> + 'object>,
```

Comment:
I don’t think there is a need to box/dyn this. I think `std::vec::IntoIter` should work.

Reply:
Oh right, I never write code like this. :sweat: